MVT/AppMVT/templates/AppMVT/languages.html

```diff
@@ -23,9 +23,9 @@ <h4 class="page-section-heading text-center text-secondary mb-0">
   {% if language %}
   <div>
     <p>{{ language }}</p>
-    <p>{{ text.text }}</p>
+    <p>{{ text }}</p>
   </div>
-  {% endif %}
+  {% else %}
```

Comment:
Coloquei esse else pra não exibir o formulário quando for renderizar o resultado da busca,
mas na verdade acho que poderia ter mantido o formulário de busca né.
Acho que não tem porque não exibir, assim o usuário pode ler a descrição de uma linguagem
e dentro da mesma página seguir pesquisando outra linguagem.